apps/aecore/src/aec_governance.erl

```diff
@@ -163,26 +165,26 @@ state_gas_per_block(oracle_response)     -> state_gas_per_block(oracle_registrat
 %% calling contract create transaction or contract call transaction)
 %% at least the gas of the call instruction (e.g. `CALL`) used
 %% for calling the primop.
-primop_base_gas(?PRIM_CALL_SPEND              ) -> 0;
-primop_base_gas(?PRIM_CALL_ORACLE_REGISTER    ) -> 0;
-primop_base_gas(?PRIM_CALL_ORACLE_QUERY       ) -> 0;
-primop_base_gas(?PRIM_CALL_ORACLE_RESPOND     ) -> 0;
-primop_base_gas(?PRIM_CALL_ORACLE_EXTEND      ) -> 0;
-primop_base_gas(?PRIM_CALL_ORACLE_GET_ANSWER  ) -> 0;
-primop_base_gas(?PRIM_CALL_ORACLE_GET_QUESTION) -> 0;
-primop_base_gas(?PRIM_CALL_ORACLE_QUERY_FEE   ) -> 0;
-primop_base_gas(?PRIM_CALL_AENS_RESOLVE       ) -> 0;
-primop_base_gas(?PRIM_CALL_AENS_PRECLAIM      ) -> 0;
-primop_base_gas(?PRIM_CALL_AENS_CLAIM         ) -> 0;
-primop_base_gas(?PRIM_CALL_AENS_UPDATE        ) -> 0;
-primop_base_gas(?PRIM_CALL_AENS_TRANSFER      ) -> 0;
-primop_base_gas(?PRIM_CALL_AENS_REVOKE        ) -> 0;
-primop_base_gas(?PRIM_CALL_MAP_EMPTY          ) -> 0;
-primop_base_gas(?PRIM_CALL_MAP_GET            ) -> 0;
-primop_base_gas(?PRIM_CALL_MAP_PUT            ) -> 0;
-primop_base_gas(?PRIM_CALL_MAP_DELETE         ) -> 0;
-primop_base_gas(?PRIM_CALL_MAP_SIZE           ) -> 0;
-primop_base_gas(?PRIM_CALL_MAP_TOLIST         ) -> 0.
+primop_base_gas(?PRIM_CALL_SPEND              ) -> 12000;
+primop_base_gas(?PRIM_CALL_ORACLE_REGISTER    ) -> 12000;
+primop_base_gas(?PRIM_CALL_ORACLE_QUERY       ) -> 12000;
+primop_base_gas(?PRIM_CALL_ORACLE_RESPOND     ) -> 12000;
+primop_base_gas(?PRIM_CALL_ORACLE_EXTEND      ) -> 12000;
+primop_base_gas(?PRIM_CALL_ORACLE_GET_ANSWER  ) -> 12000;
+primop_base_gas(?PRIM_CALL_ORACLE_GET_QUESTION) -> 12000;
+primop_base_gas(?PRIM_CALL_ORACLE_QUERY_FEE   ) -> 12000;
+primop_base_gas(?PRIM_CALL_AENS_RESOLVE       ) -> 12000;
+primop_base_gas(?PRIM_CALL_AENS_PRECLAIM      ) -> 12000;
+primop_base_gas(?PRIM_CALL_AENS_CLAIM         ) -> 12000;
+primop_base_gas(?PRIM_CALL_AENS_UPDATE        ) -> 12000;
+primop_base_gas(?PRIM_CALL_AENS_TRANSFER      ) -> 12000;
+primop_base_gas(?PRIM_CALL_AENS_REVOKE        ) -> 12000;
+primop_base_gas(?PRIM_CALL_MAP_EMPTY          ) -> 1000;
+primop_base_gas(?PRIM_CALL_MAP_GET            ) -> 1000;
+primop_base_gas(?PRIM_CALL_MAP_PUT            ) -> 1000;
+primop_base_gas(?PRIM_CALL_MAP_DELETE         ) -> 1000;
+primop_base_gas(?PRIM_CALL_MAP_SIZE           ) -> 1000;
+primop_base_gas(?PRIM_CALL_MAP_TOLIST         ) -> 6000.
```

Comment:
Why would `tolist` be 6 times more expensive than the other maps operations. Once converted to a list we will charge for its size/memory?!